scene/gui/rich_text_label.h

```diff
@@ -642,6 +667,7 @@ class RichTextLabel : public Control {
 VARIANT_ENUM_CAST(RichTextLabel::AutowrapMode);
 VARIANT_ENUM_CAST(RichTextLabel::ListType);
 VARIANT_ENUM_CAST(RichTextLabel::ItemType);
+VARIANT_ENUM_CAST(RichTextLabel::MenuItems);
```

Comment:
This likely no longer needs to be exposed since `_menu_option` isn't.

Reply:
Fixed